gtk2_ardour/ardour_ui2.cc

```diff
@@ -429,6 +429,7 @@ ARDOUR_UI::setup_transport ()
 	transport_table.set_spacings (0);
 	transport_table.set_row_spacings (4);
 	transport_table.set_border_width (0);
+	transport_table.set_size_request (0, -1);
```

Comment:
This breaks various toolbar widgets, that require the toolbar width to expand or fill available space:
e.g. the mini-timeline, Rule Solo buttons layout, etc.

![image](https://user-images.githubusercontent.com/30246/71989704-6ad02180-3232-11ea-8bb4-d75c66db257f.png)

Depending on the default font used on your system 1024x768 usually requires a scaling down to 90% in Preferences > Appearance.

Reply:
You're right